cypress/integration/MultiSelect/common/index.js

```diff
@@ -0,0 +1,7 @@
+import { Given } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a MultiSelect with no selected value', () => {
+    cy.visitStory('Testing:MultiSelect', 'Required')
+    cy.getFormValue('multiSelect').then(arg => console.log('arg', arg))
```

Comment:
Is is necessary to log to console here? I'd prefer removing this personally. If necessary devs can always add it.

Reply:
Nothing breaks when I remove it, so guess not.